### PR TITLE
Disable unused tiles only for DDNet entities

### PR DESCRIPTION
**Pull request by 12pm**
Sorry messed something up

### CODE REVIEW

File: src/game/editor/editor.h [hunk 1]
```diff
@@ -744,7 +744,7 @@ class CEditor : public IEditor
 		m_SpeedupAngle = 0;
 		m_LargeLayerWasWarned = false;
 		m_PreventUnusedTilesWasWarned = false;
-		m_AllowPlaceUnusedTiles = false;
+		m_AllowPlaceUnusedTiles = 0;
```
Comment (def-) [inline src/game/editor/editor.h]: Why is it an int now?

Comment (12pm) [reply src/game/editor/editor.h]: Need 3 states for it now: -1 = deactivated, 0 = off, 1 = on

Comment (def-) [reply src/game/editor/editor.h]: How do 0 and -1 behave differently? What about "m_AllowPlaceUnusedTiles = !m_AllowPlaceUnusedTiles;" in editor.cpp?

Comment (12pm) [reply src/game/editor/editor.h]: -1 greys out the button, the line in editor.cpp swaps it between 0 and 1

Comment (def-) [reply src/game/editor/editor.h]: Thanks for the explanation.
bors r+

**Pull request merged**